src/inject/fallback.ts

```diff
@@ -0,0 +1,8 @@
+const fallbackStyle = document.createElement('style');
+fallbackStyle.textContent = 'html,body,body :not(iframe) { background-color:#181a1b!important;border-color:#776e62!important;color:#e8e6e3!important;';
```

Comment:
Could you explain?

This is the **fallback** style that means everything just has a base color so it won't be white in your eyess. It's reset after a the dynamic styles comes.

Reply:
I mean, the base colours that are set in the browser preferences that are applied before any CSS overrides them.

Reply:
That's the problem. When the document is loaded the base colors are gone and it's switch to the website render. And the goal of darkreader is to make light pages dark so if it switches to the website it's a light flash. However with this being injected into the page it will not flash a white-ish color. Nothing is being changed the order of injecting this css is just now the first thing to do. Else it would be after some microcode that is causing the white flash.